src/Behat/Behat/Context/Environment/InitializedContextEnvironment.php

```diff
@@ -24,6 +24,8 @@
  * @see ContextEnvironmentHandler
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * @template T of Context
```

Comment:
making the `InitializedContextEnvironment` class generic does not make sense here